src/PerfViewExtensions/GlobalSrc/Global.csproj

```diff
@@ -11,24 +11,6 @@
     <AssemblyName>Global</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <!-- Determine if we are building 'next to' the main PerfView sources -->
```

Comment:
The sensibility feature was used to built extensions to PerfView by the VS Performance team.     Like most features it is hard to know who else is using it (But I have referred several people to it).  

I think non-trivial take-backs have to be done with a more careful process.   At the least we have to give people notice, and we should ideally have its replacement in hand.   The change that makes the break should not be mixed in with other things if that can be avoided.  

But first we have to understand why we are doing it.    You mention that it is getting in your way, but don't mention how.   

My recommendation is that you assume that we do need to keep it for now, and fix this pull request accordingly.      If you feel strongly that it is getting in the way, open an issue on that alone and we start a discussion on the pros and cons.   That discussion would probably also include what we think is a reasonable replacement (some part of which would probably need to be done before removing the old support).    But to avoid bogging down this issue on the sensibility issue (which make take a while to resolve),   I would simply avoid breaking it in this change.